seimas/accounts/views.py

```diff
@@ -8,16 +8,20 @@
 import seimas.accounts.helpers.allauth as allauth_helpers
 import seimas.accounts.forms as accounts_forms
 from seimas.website.helpers import formrenderer
-
+#from allauth.account.auth_backends.AuthenticationBackend import AuthenticationBackend
 
 def login(request):
+    form1 = accounts_forms.LoginForm(request.POST)
```

Comment:
Use better naming for variables. In this case `form1` could be replaced by `login_form`.